integration/RUNLOCAL

```diff
@@ -3,5 +3,9 @@
 # To use this script, drop a bzImage here and away you go.
 # Some examples:
 # ./RUNLOCAL
-# ./RUNLOCAL -v  -test.run TestTczclient
-UROOT_QEMU="qemu-system-x86_64" UROOT_KERNEL=bzImage go test "$@"
+# ./RUNLOCAL -v -test.run=TestScript
+
+export UROOT_QEMU="qemu-system-x86_64" 
```

Comment:
Add -enable-kvm as a default arg here? I expect CircleCI won't be able to do that, but most users locally should be able to?

Perhaps test if /dev/kvm exists, and add it if it does?

Reply:
Adding it as a default arg!